pkg/alpine/mirrors/3.16/main

```diff
@@ -129,6 +130,7 @@ lz4-libs
 lzo-dev
 m4
 make
+meson
```

Comment:
I see that modemmanager is using meson as build system (which depends on ninja) and I'm fine with adding this packages, but it makes me wonder how much space this different building system would take.

Reply:
Building tools do not end up in the built image, so there is not image size increase from them.

Reply:
Then we don't care how they're build :D